content/index.de.md

```diff
@@ -0,0 +1,248 @@
+- Mehr Beispiele über Vorteile für andere finden Sie in den Ressourcen zu Mobile [[Geteilte Erfahrungen im Web: gemeinsame Barrieren von Nutzer:innen mit mobilen Geräten und Menschen mit Behinderung]](/standards-guidelines/shared-experiences/), Multimedia [Genutzt von Menschen mit und ohne Behinderungen](/media/av/users-orgs/#situations) sowie in der archivierten Seite [Ein barrierefreies Web nutzt Menschen mit und ohne Behinderungen](https://www.w3.org/WAI/business-case/archive/soc#groups).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+## Barrierefreiheit ist wichtig für Einzelne, Unternehmen und Gesellschaft {#important}
+
+Das Web ist eine zunehmend wichtigere Ressource in vielen Bereichen des Lebens: Bildung, Beschäftigung, Regierung, Handel, Gesundheitsversorgung, Freizeit und mehr. Ein barrierefreies Web ist essenziell, um Menschen mit verschiedenen Fähigkeiten gleichen Zugang und gleiche Chancen zu bieten. Der Zugang zu Kommunikationstechnologien, einschließlich des Webs, wurde im Übereinkommen über die Rechte von Menschen mit Behinderungen (UN [CRPD](https://www.un.org/development/desa/disabilities/convention-on-the-rights-of-persons-with-disabilities.html)) als grundlegendes Menschenrecht festgelegt.
+
+Das Web bietet für Menschen mit Behinderungen eine noch nie dagewesene Möglichkeit für den Zugang zu Informationen sowie für die Interaktion, in dem Sinne, dass Barrieren von Print-, Audio- und Videomedien viel einfacher mittels Webtechnologien beseitigt werden können.
+
+Barrierefreiheit unterstützt die soziale Inklusion von Menschen mit Behinderungen und anderen, darunter:
+
+- älteren Menschen
+- Menschen in ländlichen Gegenden
+- Menschen in Entwicklungsländern
+
+**Es gibt auch aus unternehmerischer Sicht ein starkes Argument für Barrierefreiheit.** Wie im vorigen Abschnitt gezeigt, verbessert barrierefreies Design die allgemeine Nutzer:innenerfahrung und Befriedigung, insbesondere in einer Vielzahl von Umständen, über verschiedene Geräte hinweg und für ältere Nutzer:innen. Barrierefreiheit kann Ihre Marke stärken, Innovation fördern und Ihre Marktreichweite erhöhen.
+
+Barrierefreiheit ist in vielen Fällen **gesetzlich verpflichtend**.
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Weitere Infos zur Bedeutung von Barrierefreiheit" class="simple aside" %}
+{:/}
+
+-   Allgemeine Informationen über die geschäftlichen Vorteile finden Sie unter [[Gründe für digitale Barrierefreiheit aus unternehmerisches Sicht]](/business-case/).
+    -   Beispiele für Vorteile von [barrierefreien Audio- und Videomedien](/media/av/) finden Sie unter [Vorteile für Unternehmen](/media/av/users-orgs/#benefits).
+-   Anleitungen für die Erfassung von rechtlichen Anforderungen finden Sie im archivierten [Rechtliche Faktoren](https://www.w3.org/WAI/business-case/archive/pol).
```

Comment:
"Rechtliche Faktoren" translates into "Legal factors" according to my automatic translation tool. 

The original phrase is "Legal and Policy Factors". Can you adapt the translation to take that into account? 

The term "policy" is broader than "law". In the linked-to document, it is described as "requirements from governments and other organizations in the form of laws, policies, regulations, standards, guidelines, directives, communications, orders, or other types of documents, which have different degrees of authority in different systems."

Reply:
I see. Unfortunately I don't know how to translate "policy" other than "Richtlinie", but "Richtlinien Faktoren" is not a proper expression. Maybe something like "Rechtliche Faktoren und Richtlinien"?